examples/test.js

```diff
@@ -0,0 +1,174 @@
+/* global QUnit */
+
+'use strict';
+
+const pathImage = __dirname;
+const GONE = { status: 'gone' };
+
+const fs = require('fs');
+
+const db = require('./model-inmemory.js');
+
+const initialData = [
+  { id: 1, title: 'I caught a little fish...', file: '/img/1.png' },
+  { id: 2, title: 'The fish I caught was this big.', file: '/img/2.png' },
+  { id: 3, title: 'The fish I caught was quite big.', file: '/img/3.png' },
+  { id: 4, title: 'I caught the biggest fish you\'ve ever seen.', file: '/img/4.png' },
+];
+
+function arrayInOrder(arr, order) {
+  const retval = [];
+  for (const id of order) {
+    retval.push(arr.find((x) => {
+      return x.id === id;
+    }));
```

Comment:
just fyi, you don't need to change this here, but there's a shortcut for this: 
`retval.push(arr.find(x => x.id === id));`